### PR TITLE
Backlog 36168 7 sep cleancode

### PR DESCRIPTION
**Pull request by amitkumar3hitachi**

### CODE REVIEW

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 7m 17s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl plugins/repositories/core
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositoryOpenRecentFileExtensionPoint.java [hunk 1]
```diff
@@ -29,8 +29,7 @@
 import org.pentaho.di.core.logging.LogChannelInterface;
```
Comment (peterrinehart) [inline plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositoryOpenRecentFileExtensionPoint.java:29]: Copyright date on this file needs to be updated.

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositorySpoonStartExtensionPoint.java [hunk 2]
```diff
@@ -33,8 +33,7 @@
 import org.pentaho.di.pan.CommandLineOption;
 import org.pentaho.di.repository.RepositoryMeta;
 import org.pentaho.di.ui.repo.controller.RepositoryConnectController;
-import org.pentaho.di.ui.repo.dialog.RepositoryConnectionSWT;
```
Comment (peterrinehart) [inline plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositorySpoonStartExtensionPoint.java:36]: Copyright date on this file needs to be updated.

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RequestLoginToRepositoryExtensionPoint.java [hunk 3]
```diff
@@ -30,7 +30,7 @@
 import org.pentaho.di.repository.Repository;
```
Comment (peterrinehart) [inline plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RequestLoginToRepositoryExtensionPoint.java:30]: Copyright date on this file needs to be updated.

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java [hunk 4]
```diff
@@ -29,7 +29,7 @@
 import org.pentaho.di.i18n.BaseMessages;
```
Comment (peterrinehart) [inline plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java:29]: Copyright date on this file needs to be updated.

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java [hunk 5]
```diff
@@ -80,9 +80,11 @@ public Object handle( Object objectToHandle, Throwable exception, Method method,
 
   boolean showLoginScreen( RepositoryConnectController repositoryConnectController ) {
     String message = BaseMessages.getString( PKG, "Repository.Reconnection.Message" );
-    RepositoryDialog loginDialog = new RepositoryDialog( getSpoon().getShell(), repositoryConnectController );
+    RepositoryConnectionDialog loginDialog = new RepositoryConnectionDialog( getSpoon().getShell());
     RepositoryMeta repositoryMeta = repositoryConnectController.getConnectedRepository();
-    return loginDialog.openRelogin( repositoryMeta, message );
+    repositoryConnectController.setRelogin( true );
+    return loginDialog.createDialog( repositoryMeta.getName() );
+    //    return loginDialog.openRelogin( repositoryMeta, message );
```
Comment (peterrinehart) [inline plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java:87]: Remove commented code.

File: plugins/repositories/core/src/test/java/org/pentaho/di/ui/repo/controller/RepositoryConnectControllerTest.java [hunk 6]
```diff
@@ -188,18 +188,18 @@ public void testConnectToRepository() throws Exception {
     verify( repository ).connect( null, null );
```
Comment (peterrinehart) [inline plugins/repositories/core/src/test/java/org/pentaho/di/ui/repo/controller/RepositoryConnectControllerTest.java:188]: Copyright date on this file needs to be updated.

**Review by peterrinehart — CHANGES_REQUESTED** [review]
Need to address the comments, plus squash these three commits into one with an appropriate message describing the work done, and the jira tag.

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositoryOpenRecentFileExtensionPoint.java (hunk 1 above)
Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositoryOpenRecentFileExtensionPoint.java:29]: fixed.

Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositoryOpenRecentFileExtensionPoint.java:29]: fixed.

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositorySpoonStartExtensionPoint.java (hunk 2 above)
Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RepositorySpoonStartExtensionPoint.java:36]: fixed

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RequestLoginToRepositoryExtensionPoint.java (hunk 3 above)
Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/extension/RequestLoginToRepositoryExtensionPoint.java:30]: fixed

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java (hunk 4 above)
Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java:29]: fixed

File: plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java (hunk 5 above)
Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/main/java/org/pentaho/di/ui/repo/timeout/SessionTimeoutHandler.java:87]: fixed

File: plugins/repositories/core/src/test/java/org/pentaho/di/ui/repo/controller/RepositoryConnectControllerTest.java (hunk 6 above)
Comment (amitkumar3hitachi) [reply plugins/repositories/core/src/test/java/org/pentaho/di/ui/repo/controller/RepositoryConnectControllerTest.java:188]: fixed

**Pull request merged**